bedrock/base/templates/macros-protocol.html

```diff
@@ -79,11 +79,11 @@
         {{ high_res_img(image_url, {'alt': image_alt, 'l10n': l10n_image, 'class': 'mzp-c-split-media-asset'}) }}
```

Comment:
```suggestion
        {{ high_res_img(image_url, {'alt': image_alt, 'l10n': l10n_image, 'class': 'mzp-c-split-media-asset', 'loading': loading}) }}
```

Let's add this to the `high_res_img` helper too. Thanks!

Reply:
Will do.